### ISSUE TITLE
Can't execute NSL

### ISSUE THREAD

**Issue by ILKA-KZ** [moraroy/NonSteamLaunchers-On-Steam-Deck#541]
Hello, I am new to Steam Deck and Linux, and I can't figure out how to install the NSL. 
I have a Decky Loader, but can't find the NSL (I changed to "testing" but still don't see it).

So I've downloaded the desktop version, but I can't execute it. 
When I double-click the file, I get a black window pop-up for less than a second and that's it. 

Can you please explain what I am doing wrong?

**Comment by moraroy** [comment]
Please use the NSL plugin.deaktop https://github.com/moraroy/NonSteamLaunchersDecky/releases/tag/v1.0.2 once downloaded double click it.

**Comment by ILKA-KZ** [comment]
> Please use the NSL plugin.deaktop https://github.com/moraroy/NonSteamLaunchersDecky/releases/tag/v1.0.2 once downloaded double click it.

Thats what I'm doing. But when I double-click a black window appears for less than a sec. Then nothing. 
P.S. I have the decky loader

I tried to find a fix on reddit, here and youtube, only found some people getting the same thing, but no solution.

**Comment by moraroy** [comment]
@ILKA-KZ if there is a ".download" in the file name from chrome please delete it. Also please move the .desktop file to the actual desktop and try to run it from there. you can also right click it and "run it from Konsole" as well. let me know if that fixes the "black screen".

**Comment by ILKA-KZ** [comment]
@moraroy Hi there. For some reason, everytime I download the file, it has ".download" in the file name (I use Firefox). Ive tried downloading the Source Code and moving a .desktop file to the actual desktop, then clicked twice, but nothing happens. I also do not have a "run it from Konsole" on my Steam Deck. 

I moved to China, so it could internet issues, but downloading Decky Loader from Github was no issue for me.

Do you have any ideas on what should I do?

**Comment by moraroy** [comment]
@ILKA-KZ what about "run in Konsole" does that effect anything?

**Comment by murderuo** [comment]
hello from Turkey. i downloaded this plugin and move the file Desktop then i run my downloaded file but my steam restarting. 

are you seeing before this error ?

**Comment by moraroy** [comment]
@murderuo this is normal behavior. Steam restarted so it can add the shortcut into your library. You should see your downloaded launcher in your Steam library.

**Comment by moraroy** [comment]
@murderuo are you using the NonSteamLaunchers.desktop? Or the NSLPlugin.desktop? The plugin version will restart into game mode. Make sure you have decky loader installed first.

**Comment by murderuo** [comment]
Yes i installed decky_installer first. after that  i clicked DOWNLOAD BUTTON  and i download NSLplugin.desktop the i click and my device is restart. Ok understand where i see thle plugin game mode ?

**Comment by moraroy** [comment]
@murderuo if you installed decky loader correctly and the NSLPlugin.Desktop correctly, in Game mode, press the three dots on your Steam Deck (...) this will open the side panel and you will see a 🔌  icon. This has your plugins that you installed including NonSteamLaunchers.

**Comment by murderuo** [comment]
ok . i can see now. thanks. but i have diffrent problem now. i can see nonsteamlaunchers in decky, i click the i see install menu. but nothing happend. only B button is working fine. why is happened ?

**Comment by murderuo** [comment]
> ok . i can see now. thanks. but i have diffrent problem now. i can see nonsteamlaunchers in decky, i click the i see install menu. but nothing happend. only B button is working fine. why is happened ?

on, i got it . i uninstalled decky and install new version again. then i see nonSteamLauncher plugin, i click this plugin and i can install launchers. Especialy EA launcher working fine but Epic Games Launcher doesnt working. 

i use Heroic launcher for Epic Games Launcher , i recommend this way other users. maybe you can review how they fix tihs ?

**Comment by moraroy** [comment]
@murderuo yes Epic broke recently, as seen here you can fix, you have to delete some files  click here #513
